rewrite-xml/src/main/java/org/openrewrite/xml/tree/Xml.java

```diff
@@ -465,6 +465,13 @@ public String getPrefix() {
             public String toString() {
                 return "</" + name + ">";
             }
+
+            public boolean isSelfClosing(Xml.Tag tag) {
```

Comment:
Given how small this method is, and for now only used in the visitor, I'd pull this method to the visitor instead. That helps limit our publicly visible API & what we have to support going forward.